build-tests/api-extractor-scenarios/etc/spanSorting/api-extractor-scenarios.api.json

```diff
@@ -375,27 +375,45 @@
           "implementsTokenRanges": []
         },
         {
-          "kind": "Variable",
-          "canonicalReference": "api-extractor-scenarios!exampleD:var",
+          "kind": "Function",
+          "canonicalReference": "api-extractor-scenarios!exampleD:function(1)",
           "docComment": "/**\n * Outer description\n *\n * @public\n */\n",
           "excerptTokens": [
             {
               "kind": "Content",
-              "text": "exampleD: "
+              "text": "exampleD: (o: "
             },
             {
               "kind": "Content",
-              "text": "(o: {\n    a: number;\n    b(): string;\n}) => void"
+              "text": "{\n    a: number;\n    b(): string;\n}"
+            },
+            {
+              "kind": "Content",
+              "text": ") => "
+            },
+            {
+              "kind": "Content",
+              "text": "void"
             }
```

Comment:
Why did this get split up in a different way?

Reply:
The `excerptTokens` are processed differently for `function` kinds to be able to map the tokens back to the `parameterTypeTokenRange` and `returnTypeTokenRange`